module-10/10.2/client/package.json

```diff
@@ -16,7 +14,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest --coverage",
```

Comment:
React-scripts already includes jest library; we may not need to replace the default `test` command here.

see docs: https://create-react-app.dev/docs/running-tests/